src/core/Akka/Helios.Concurrency.DedicatedThreadPool.cs

```diff
@@ -94,11 +94,16 @@ internal class DedicatedThreadPoolSettings
                 throw new ArgumentOutOfRangeException(nameof(numThreads), $"numThreads must be at least 1. Was {numThreads}");
         }
 #else
-        public DedicatedThreadPoolSettings(int numThreads, string name = null, TimeSpan? deadlockTimeout = null)
-            : this(numThreads, DefaultThreadType, name, deadlockTimeout)
+        public DedicatedThreadPoolSettings(int numThreads, string name = null, TimeSpan? deadlockTimeout = null, SynchronizationContext synchronizationContext = null)
+            : this(numThreads, DefaultThreadType, name, deadlockTimeout, synchronizationContext)
```

Comment:
Go ahead and remove `SynchronizationContext` from this. Not used.

Reply:
I missed that one, good catch